### PR TITLE
Add: Button component with styling and html

### PR DESCRIPTION
**Pull request by Logan1x**

### CODE REVIEW

**Review by thegeorgenikhil — CHANGES_REQUESTED** [review]
` border-radius: 0.2rem;
  padding: 0.5rem;
  margin: 0.5rem;
  font-size: 1rem;
  font-weight: bold;
  border: none;
  cursor: pointer;`

Hey Kushal, it would be better it could  make a separate class .btn and add these repeating properties in both the buttons and add that .btn class tho those button elements
It would make it more reusable and easier to customize in future when you are adding more buttons

**Review by thegeorgenikhil — COMMENTED** [review]
Everything else look fine!✅

File: button/button.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+.btn-section {
+  display: grid;
+  place-content: center;
+  place-items: center;
+}
+
+.btn-primary {
+  background-color: var(--primary-btn-color);
+  color: white;
```
Comment (Parul1999) [inline button/button.css]: use css variable

**Review by Parul1999 — CHANGES_REQUESTED** [review]
Use CSS variables, instead of hard-coding the color code.

File: button/button.css (hunk 1 above)
Comment (Logan1x) [reply button/button.css]: Done, replaced with color variable :)

**Comment by Logan1x** [comment]
Thanks Nikhil & Parul for your suggestions, merging this now :)

**Pull request merged**